samples/boards/96b_argonkey/sensors/README.rst

```diff
@@ -40,7 +46,7 @@ Building and Running
 ********************
 
  .. zephyr-app-commands::
-    :zephyr-app: samples/boards/96b_argonkey
+    :zephyr-app: samples/boards/96b_argonkey/sensors
```

Comment:
The ``.. zephyr-app-command`` directive above should not be indented by a space.